builds/fabric8-che/plugins/keycloak-plugin-server/src/main/java/com/redhat/che/keycloak/server/KeycloakAuthenticationFilter.java

```diff
@@ -41,7 +42,10 @@
     private boolean keycloakDisabled;
 
     @Inject
-    private KeycloakUserChecker userChecker;
+    private KeycloakTokenStore         kcTokenStore;
```

Comment:
Field injection requires the use of some injection framework when writing tests. Personally, I'm not a fan. Why not use constructor injection? Is there an opinion on the Che team?